pkg/instancetype/instancetype.go

```diff
@@ -852,6 +854,38 @@ func (m *InstancetypeMethods) inferDefaultsFromDataVolumeSourceRef(sourceRef *v1
 	return "", "", fmt.Errorf("unable to infer defaults from DataVolumeSourceRef as Kind %s is not supported", sourceRef.Kind)
 }
 
+func applyInstanceTypeAnnotations(field *k8sfield.Path, annotations map[string]string, target metav1.Object) (conflicts Conflicts) {
+	if target.GetAnnotations() == nil {
+		target.SetAnnotations(make(map[string]string))
+	}
+
+	existingAnnotations := target.GetAnnotations()
+	for key, value := range annotations {
+		if val, exists := existingAnnotations[key]; exists && val != value {
+			// TODO? return Conflicts{field.Child(key)}
```

Comment:
Returning all of the conflicts is better UX IMHO so I wouldn't change this to return the first.

Reply:
Ok, I will keep it like this